pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -30,6 +30,7 @@ var (
 	// defaultCleanOption is default clean option
 	defaultCleanOption = CleanOption{
 		PageSize:          10000,
+		RetryCount:        3,
```

Comment:
Set it to 10 as we want to delete the files as much as possible?

Reply:
the backoff of the clean job is 5, so the total retry count is 15, so I think it is enough. And clients can modify the option if needed

Reply:
But it's more lightweight to retry in the current Pod instead of waiting for another Pod, right?

Reply:
default to 5